CODEOWNERS

```diff
@@ -128,6 +128,9 @@
 # renovate groupName=oci-run-script
 /task/run-script-oci-ta @konflux-ci/build-maintainers @Zokormazo @arewm
 
+# renovate groupName=ko-builder
+/task/ko-oci-ta @konflux-ci/build-maintainers @gallen @vdemeester @arewm
```

Comment:
All owners should be within the org, looks like it's not the case and GitHub reports invalid CODEOWNERS file.